tools/github-actions/cascading/readme.md

```diff
@@ -1,6 +1,7 @@
 # Cascading action
 
-> **Warning**: This action is deprecated, please install the [Otter Cascading App](https://github.com/apps/otter-cascading) instead.
+> [!CAUTION] Deprecated Action
```

Comment:
not working

Reply:
Sadly yes, it was working on the VsCode preview